cmd/fetchd/cmd/genesis-asi-upgrade.go

```diff
@@ -496,6 +496,14 @@ func ASIGenesisUpgradeASISupply(jsonData map[string]interface{}, networkInfo Net
 	// add the additional coins to the overflow address balance
 	overflowAddressBalanceCoins = overflowAddressBalanceCoins.Add(additionalSupplyCoin)
 
+	// add the new supply mint record to the manifest
+	mintRecord := ASIUpgradeSupplyMint{
+		LandingAddress: supplyInfo.UpdatedSupplyOverflowAddr,
+		Amount:         sdk.NewCoins(additionalSupplyCoin),
+		NewSupplyTotal: sdk.NewCoins(newSupplyCoins),
+	}
+	manifest.SupplyMint = append(manifest.SupplyMint, mintRecord)
```

Comment:
As per my inline comments above:
```suggestion
	manifest.Supply = supplyRecord
```